jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -64,6 +64,7 @@ public final class StackWalker {
 	/*[IF JAVA_SPEC_VERSION >= 22]*/
 	private static final int J9_DROP_METHOD_INFO       = 0x10;
 	/*[ENDIF] JAVA_SPEC_VERSION >= 22 */
+	private static final int J9_GET_CALLER_CLASS       = 0x20;
```

Comment:
Is this JDK22+ specific? Should it be wrapped with `/*[IF JAVA_SPEC_VERSION >= 22]*/`?